blog/content/post/2024-07-23-new-mtl-releases.md

```diff
@@ -0,0 +1,329 @@
+![UEFI Secure Boot](/img/secure_boot.png)
+
+### [Automatic Embedded Controller update](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio/#ecr031001-ec-firmware-sync-in-coreboot)
+
+The feature allows updating the Embedded Controller firmware easily
+alongside updating the bios firmware.
+Check out the link in the section title for more details.
+
+### [Firmware update mode](https://docs.dasharo.com/guides/firmware-update/#firmware-update-mode)
+
+The Firmware Update Mode is a quick way of ensuring your device
+is prepared for a firmware update. It temporarily changes the
+configuration and returns it to the previous state after
+the update is completed. Check the link in the header for more details
+and instructions on how to use it.
+![Firmware Update Mode Confirmation Screen](/img/setup_menu_fum_confirmation.png)
+
+### [BIOS boot medium write-protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+
+The recovery bios chip on your device is write-protected by default.
+Modifying the firmware is only possible by entering the setup menu
+and disabling the write protection explicitly or implicitly by using
+Firmware Update Mode. Check the link in the header for more details
+about Dasharo Security Options.
+
+![BIOS boot medium write-protection](/img/bios_lock.png)
+
+### [SMM BIOS write protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+
+With this option enabled, the BIOS can only be modified by
+System Management Mode privileged code. It prevents flashing
+the firmware with programs like [flashrom](https://www.flashrom.org/).
+![SMM BIOS write protection](/img/smm_bios_write_protection.png)
+
+### [Early boot DMA protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+
+The IOMMU DMA protection enabled early
+in the POST process prevents Direct Memory Access attacks.
+Leaking secrets and injecting malware using devices such
+as PCIe cards and USB4/thunderbolt devices is prevented
+thanks to this feature.
+
+### [Early Sign of Life display output](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/347-sign-of-life/)
+
+The Early Sign of Life display is a feature that displays the
+firmware version during the boot phase. It is useful for
+diagnosing boot issues and verifying that the firmware
+is up-to-date.
+
+### [Current limiting for USB-PD power supplies](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31H-usb-type-c/#utc020001-usb-type-c-pd-current-limiting-ubuntu-2204)
+
+Limiting the current draw from USB-Power Delivery is an
+important safety feature that controls the maximum current
+that can flow through the USB-PD. This prevents damaging the device
+as well as what it is connected to in case of any malfunction
+like short-circuiting the USB port.
```

Comment:
Not really, this is just about respecting the limits that the USB-C charger tells us. Before, the laptop would just pull as many watts as it wanted from a charger, overdrawing from it. Most chargers would engage overcurrent protection at this point and reset or shutdown, but some cheaper chargers could be damaged if this was done repeatedly.

Reply:
[5d83859](https://github.com/3mdeb/news-and-ideas/pull/598/commits/5d838599cca210aea0fc9571724be1a9c17bb9b1)